tokio-trace-fmt/src/filter.rs

```diff
@@ -149,8 +150,19 @@ impl Filter for EnvFilter {
                     let in_span = ctx
                         .visit_spans(|_, span| {
                             if span.name() == desired {
-                                // Return `Err` to short-circuit the span visitation.
-                                Err(())
+                                if let Some((ref field, ref val)) = directive.in_field_value {
+                                    let fields = span.fields();
+                                    let matcher = format!("{}={:?}", field, val);
```

Comment:
prob could make this a lot better just wanted to get it working for my use case 😅

Reply:
Having to allocate here is definitely quite unfortunate. This is a consequence of how span fields are currently represented (as a single string), which is primarily a performance optimization. We may want to reconsider that choice to make this use case a little more efficient.